x-pack/plugins/apm/public/components/app/service_groups/service_groups_list/index.tsx

```diff
@@ -21,11 +22,56 @@ import { ServiceGroupsListItems } from './service_groups_list';
 import { Sort } from './sort';
 import { RefreshServiceGroupsSubscriber } from '../refresh_service_groups_subscriber';
 import { getDateRange } from '../../../../context/url_params_context/helpers';
+import { RedirectTo } from '../../../routing/redirect_to';
+import { ServiceGroupsTour } from '../service_groups_tour';
+import { useServiceGroupsTour } from '../use_service_groups_tour';
 
 export type ServiceGroupsSortType = 'recently_added' | 'alphabetical';
 
 export function ServiceGroupsList() {
+  const { tourEnabled, dismissTour } = useServiceGroupsTour(
+    'serviceGroupsAllServices'
+  );
+  const buttonGroupOptions = {
+    allServices: {
+      id: 'allServices',
+      label: (
+        <ServiceGroupsTour
+          tourEnabled={tourEnabled}
+          dismissTour={dismissTour}
+          title={i18n.translate(
+            'xpack.apm.serviceGroups.tour.serviceGroups.title',
+            { defaultMessage: 'View all services' }
+          )}
+          content={i18n.translate(
+            'xpack.apm.serviceGroups.tour.serviceGroups.content',
+            {
+              defaultMessage:
+                "Now that you've created a service group, you can go back and view all services here.",
```

Comment:
I wonder if this tour element is even needed anymore. With a button title of `View all services`, I don't think the tour text adds anything that the button doesn't already communicate.